dynawo/sources/Common/DYNExecUtils.cpp

```diff
@@ -77,11 +96,27 @@ void parentProcess(int fd[2], std::stringstream & ss) {
   if (strbuf.size() != 0)
     ss << strbuf << std::endl;
 }
+#endif
+
 void
-executeCommand(const std::string & command, std::stringstream & ss) {
+executeCommand(const std::string & command, std::stringstream & ss, const std::string & start_dir) {
   ss << "Executing command : " << command << std::endl;
+
+#ifdef _MSC_VER
+  ps::ipstream ips;
+  ps::child child(command, ps::shell, ps::start_dir(start_dir == "" ? "." : start_dir), (ps::std_out & ps::std_err) > ips);
+
+  string line;
```

Comment:
I don't know, it should work for both but I will have to test it because I know that pipes can cause problems... It will be adressed in another commit

Reply:
OK